database/src/main/resources/changesets/backend_KV_Store.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="rmunshi" id="BACKEND_KV_STORE">
+        <comment>
+            Storage for all Backend K/V Pairs.
+        </comment>
+        <createTable tableName="BACKEND_KV_STORE">
+            <column autoIncrement="true" name="BACKEND_KV_STORE_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_BACKEND_KV_STORE_ID"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CALL_FQN" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CALL_INDEX" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CALL_ATTEMPT" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="BACKEND_JOB_KEY" type="VARCHAR(255)">
```

Comment:
This name bugs me as it sounds very close to `BackendJobDescriptorKey` or `JobKey` but really it's just the K in a KV pair.  Maybe `STORE_KEY` (and `STORE_VALUE`)?